website/docs/r/managed_application_definition.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+subcategory: "ManagedApplication"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_application_definition"
+description: |-
+  Manages a Managed Application Definition.
+---
+
+# azurerm_managed_application_definition
+
+Manages a Managed Application Definition.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_managed_application_definition" "example" {
+  name                = "example-managedapplicationdefinition"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  lock_level          = "ReadOnly"
+  package_file_uri    = "https://github.com/Azure/azure-managedapp-samples/raw/master/Managed Application Sample Packages/201-managed-storage-account/managedstorage.zip"
+  display_name        = "TestManagedApplicationDefinition"
+  description         = "Test Managed Application Definition"
+
+  authorization {
+    service_principal_id = data.azurerm_client_config.current.object_id
+    role_definition_id   = "a094b430-dad3-424d-ae58-13f72fd72591"
```

Comment:
Use `azurerm_role_definition` instead to retreive id.

Reply:
updated